src/client/src/rt-platforms/openFin/adapter/launcherUtils.ts

```diff
@@ -0,0 +1,8 @@
+export async function isParentAppOpenfinLauncher() {
+  const app = await fin.Window.getCurrent()
```

Comment:
@pmilla1606  should we add an extra check for `fin` here, since it’s not always called from an OpenFin context?